config/routes.rb

```diff
@@ -8,15 +8,15 @@
   Hydra::BatchEdit.add_routes(self)
   # This must be the very last route in the file because it has a catch-all route for 404 errors.
     # This behavior seems to show up only in production mode.
-  mount Sufia::Engine => '/'
+  mount Sufia::Engine => '/admin'
 
   resources :searches
 
-  get "public_about" => 'cms_pages#show', id: 'about_page'
+  get "about" => 'cms_pages#show', id: 'about_page'
   get "policies" => 'cms_pages#show', id: 'policies_page'
 
   # For editing via the Sufia backend
   get "admin/policies" => 'pages#show', id: 'policies_page'
 
-  root to: 'homepage#index'
+  root to: 'searches#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.